pure-pulsars/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12.4-slim
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
```

Comment:
Can we move the `COPY . .` part here and remove this line?

Reply:
Ah, this seems to be the correct approach due to how the layers work with building a container.